src/components/ui4/scenes/HomeSceneUi4.tsx

```diff
@@ -139,11 +199,19 @@ export const HomeSceneUi4 = (props: Props) => {
               />
             </EdgeAnim>
           </>
+          {promos == null || promos.length === 0 ? null : (
+            <EdgeAnim style={{ height: theme.rem(11.5) }} enter={{ type: 'fadeInUp', distance: 30 }}>
+              <CarouselUi4 height={theme.rem(10)} width={screenWidth}>
+                {promos.map(promoInfo => (
+                  <PromoCardUi4 navigation={navigation} promoInfo={promoInfo} key={promoInfo.localeMessages[0]} />
```

Comment:
Shouldn't this be `localeMessages[language]` or something like that

Reply:
It's just the key and we're guaranteed the first `localeMessage`

Reply:
But we shouldn't be using the first message. We should be using the one that matches the language.

Reply:
We should be using the language not the first entry. That's why we even have a localeMessage in the first place.